lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -3,6 +3,7 @@ module Helper
     class AppcenterHelper
 
       # create request
+      # TODO: handle dsym vs mapping
```

Comment:
Does this mean there is still functionality missing?

Reply:
no, I think it is just old comment